bip32/src/error.rs

```diff
@@ -15,6 +15,19 @@ pub enum Error {
     /// BIP39-related errors.
     Bip39,
 
+    /// BIP39 can't find the word in the selected wordlist.
+    Bip39InvalidWord,
+
+    /// BIP39-related errors.
+    Bip39InvalidPhraseSize,
+
+    ///BIP39 entropy must have 16 or 32 bytes
+    Bip39InvalidEntropySize,
+    
+    ///BIP39 invalid checksum
+    Bip39InvalidChecksum,
```

Comment:
Instead of making several `Bip39`-prefixed variants, I think it would be better to have `Bip39(bip39::Error)` and a subdivided error type concerned with just those failure modes, even if that's a breaking change.